README.md

```diff
@@ -55,13 +55,17 @@ import { I18nModule } from 'nestjs-i18n';
 
 @Module({
   imports: [
-    I18nModule.forRoot({path: path.join(__dirname, '/i18n/'), fallbackLanguage: 'en'}),
+    I18nModule.forRoot({
+      path: path.join(__dirname, '/i18n'), 
+      filePattern: '*.json',
```

Comment:
I think about removing the filepattern and rename the path property to selector or something like that. Then the selector is the path and pattern in one. (`i18n/**/*.json`)

Reply:
ok, sure.. i'll rework it ;)

Reply:
oh wait.. no.. we def. need both params, because otherwise you cannot resolve the folder-structure, because it is lost through the glob..

Reply:
True :p then let's do it like this indeed, a path and a filePattern. But this would implement a breaking change so can you make a pull request to the v4 branch?